hs628_tools/Test.py

```diff
@@ -38,26 +38,49 @@ def test_train():
             # data_len =sum(1 for line in data)
             # print data_len
 
-
+        clean_data = []
         for column in data:
 
-            for column in data:
+            for row in column:
+
+                if row == "?":
+                    pass
+                if row == "NA":
+                    pass
+                if row == "NAN":
+                    pass
+
+                else:
+                    clean_data.append(row)
 
-                for row in column:
 
-                    if row == "?":
-                        pass
-                    if row == "NA":
-                        pass
-                    if row == "NAN":
-                        pass
-                    else:
-                        print (row)
-            return row
+        return clean_data
 
         #********
 
     test_train()
 
-    # row_len = sum(1 for line in test_train())
-    # print row_len
+    row_len = sum(1 for line in test_train())
+    print row_len
+
+
+    def split_ratio(row_len):
+        """
+
+        :return: data split ratio based on the number of data points
+        """
+        if row_len <= 500:
```

Comment:
How are these metrics being determined? Do you have reasoning as to why you chose these upper and lower quartiles?